### ISSUE TITLE
Oncomplete Video

### ISSUE THREAD

**Issue by Hizbullah** [VodkaBears/Vide#153]
Hi, how to call a function when a video is played once, i added loop: false so the video just play once and i want to fire a function when its completely played.

Thanks.

**Comment by VodkaBears** [comment]
There is `getVideoObject` method: https://github.com/VodkaBears/Vide#methods
There is `HTMLVideoElement` documentation: https://developer.mozilla.org/en/docs/Web/API/HTMLVideoElement

**Comment by VodkaBears** [comment]
See events.

**Comment by Hizbullah** [comment]
Thank you so much for your replay, unfortunately I didn't find such events for oncomplete video.

**Comment by VodkaBears** [comment]
https://developer.mozilla.org/en-US/docs/Web/Events/ended

**Comment by Hizbullah** [comment]
Yeah i tried this event before i posted the question but it didn't help me at all :(

**Comment by chrisrowe** [comment]
`
var video = $('.js-video').data('vide').getVideoObject();
video.onended = function() {
	alert("The video has ended");
};
`

Works great for me when looping is disabled.
